source/index.js

```diff
@@ -36,6 +37,27 @@ var initWidget = function (node, options) {
     return widget;
 };
 
+var initLikely = function (nodes, options) {
+    var realNodes;
+    if (Array.isArray(nodes)) {
+        // An array of nodes was passed
+        realNodes = nodes;
+    } else if (nodes instanceof Node) {
+        // A single node was passed
+        realNodes = [nodes];
+    } else {
+        // Options were passed, or null/undefined was passed, or the function was called without arguments
+        // Find and initialize all existing Likely widgets
+        realNodes = dom.findAll('.' + config.name);
+    }
+
+    var realOptions = options || nodes || {};
```

Comment:
Wow, thanks a lot for noticing this, it could’ve made to the release unnoticed and then break stuff for many people. Will fix this.